js/pages/incidence-rates/components/iranalysis/components/results.js

```diff
@@ -86,6 +86,10 @@ define([
 			});
 
 			this.executionDisabledReason = ko.computed(() => this.dirtyFlag().isDirty() ? constants.disabledReasons.DIRTY : constants.disabledReasons.ACCESS_DENIED);
+
+			this.disableExportAnalysis = ko.pureComputed(() => {
+				return this.dirtyFlag().isDirty() || this.sources().filter(s => s.info() != null).length === 0;
```

Comment:
pls, use strict comparison, but `!!s.info()` looks better in such case,
moreover why not smth like `this.sources.some(s => !!s.info())`?

Reply:
updated